todo/src/components/Todo.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import {toggleCompleted, deleteItem} from '../actions';
+import {connect} from 'react-redux';
+
+const DivWrapper = styled.div`
+  display: flex;
+  padding: 10px;
+`;
+
+const DeleteButton = styled.button`
+  padding: 20px;
+  border-radius: 50%;
+  background-color: black;
+  border: 1px solid black;
+  color: red;
+
+  &:hover {
+    background-color: red;
+    color: black;
+  }
+`;
+
+const DivTodo = styled.div`
+  margin-right: 10px;
+  padding: 15px;
+  font-size: 20px;
+  border-radius: 5px;
+  background-color: #3c963c;
+  color: ${props => (props.complete ? '#1b1b1b' : '#ffffff')};
```

Comment:
I really like the styling you did on your todo app, it's unique and stands out. Good job using styled-components to do it.